modules/effects/spec/types/effects_module.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { expecter } from 'ts-snippet';
+import { compilerOptions } from './utils';
+
+describe('EffectsModule()', () => {
+  const expectSnippet = expecter(
+    (code) => `
+      import { EffectsModule } from '@ngrx/effects';
+
+      class EffectFixture {}
+
+      ${code}`,
+    compilerOptions()
+  );
+
+  describe('forRoot', () => {
+    it('should compile without params', () => {
+      expectSnippet(`
+        EffectsModule.forRoot();
+      `).toSucceed();
+    });
+
+    it('should compile with a single param', () => {
+      expectSnippet(`
+        EffectsModule.forRoot(EffectFixture);
+      `).toSucceed();
+    });
+
+    it('should compile with an array param', () => {
+      expectSnippet(`
+        EffectsModule.forRoot([EffectFixture]);
+      `).toSucceed();
+    });
+
+    it('should compile with a spreaded array param', () => {
+      expectSnippet(`
+        EffectsModule.forRoot(EffectFixture, EffectFixture);
+      `).toSucceed();
+    });
+  });
+
+  describe('forFeature', () => {
+    it('should compile without params', () => {
+      expectSnippet(`
+        EffectsModule.forFeature();
```

Comment:
Question: Why would we allow this?

Reply:
This was previously also supported because we provided an empty array as default argument

https://github.com/ngrx/platform/blob/43198a2fa8d11faa3e4cc080b9468bac385b002b/modules/effects/src/effects_module.ts#L44

Reply:
I see. Thanks